Adding a new section including the tools requirements to the docs (#2)
snapcore/snapd-testing-tools: README.md

For news and updates, follow us on [Twitter](https://twitter.com/snapcraftio)
and on [Facebook](https://www.facebook.com/snapcraftio).

## Adding new tools

The tools included in this project are intended to be reused by other projects.

Read the following considerations before adding new tools:

 - Each tool has to be stored in the tools directory

>  - At least 1 spread test needs to be included in the tests directory for each tool

Reviewer: ```suggestion
 - Each tool needs to be accompanied by at least 1 spread test in `tests/<tool-name>/`
```

Author: awesome, thanks, all changes applied

>  - In case the tool is a shell script, this needs to pass the shellcheck

Reviewer: ```suggestion
 - If the tool is a shell script, it needs to first pass a [ShellCheck](https://github.com/koalaman/shellcheck) assessment
```

>  - The tools need to be as generic as possible

Reviewer: ```suggestion
 - All tools need to be as generic as possible
```

>  - A command line interface (CLI) has to be provided by the tool. including a help

Reviewer: ```suggestion
- Each tool must also provide a command line interface (CLI), including _help_ output
```
